locator/topology.hh

```diff
@@ -55,12 +55,18 @@ public:
      */
     bool has_endpoint(inet_address, pending with_pending = pending::no) const;
 
+    const std::unordered_set<inet_address>&
+    get_datacenter_endpoints(const sstring& dc) const;
```

Comment:
nit: std::string_view will avoid constructing an sstring if the caller only has a string_view.